tests/test_serialisation.py

```diff
@@ -174,6 +175,45 @@ def test_deserialise():
         deserialise({"type": "NoEntry"})
 
 
+def test_deserialise_total_price():
```

Comment:
It would be a bit more concise to add these cases to the existing test for the deserialise_posting function. Will do this in the merge.